astropy/units/tests/test_quantity_ufuncs.py

```diff
@@ -711,20 +711,16 @@ def test_onearg_test_ufuncs(self, ufunc):
         assert out.dtype == bool
         assert np.all(out == ufunc(q.value))
 
+    # Ignore RuntimeWarning raised on Windows and s390.
+    @pytest.mark.filterwarnings('ignore:.*invalid value encountered in sign')
```

Comment:
I thought we agreed in #7928 to not blindly ignore this warning as such so we'll know when Numpy fixes its bug upstream? Isn't this a step backward?

Reply:
Yes, that was when it was windows only... now I just don't know anymore where it could fail, so decided we might as well ignore it, as it really isn't our problem. (But I admit to being too lazy to check how to find out all possible places where this could be wrong...)

Reply:
Hey, it turns out I now get these warnings even on my linux box (for numpy-dev), so filtering is OK after all. How nice to have laziness rewarded!

Reply:
Hah! Okay, then.